package.json

```diff
@@ -52,6 +53,7 @@
     "eslint-plugin-solid": "0.12.1",
     "eslint-plugin-tsdoc": "0.2.17",
     "fs-extra": "11.1.1",
+    "glob": "^10.2.1",
```

Comment:
Can you explain why? pin? is that for allowing dependabot to do its magic?

Reply:
General good practice in case something happens, as having the ^ means auto-updating to newer non-major versions without any approval. Also repeatability of builds but that doesnt really matter for this dependency.